### PR TITLE
Additional fields for job application

### PR DESCRIPTION
**Pull request by lukasz-kaniowski**

### CODE REVIEW

File: assets/js/main.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+$(document).ready(() => {
```
Comment (gjvis) [inline assets/js/main.js]: Why wait for document ready? The JS is injected at the end of the body, so the dom will be ready for manipulation at this point

File: assets/js/main.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  // Use GOV.UK selection-buttons.js to set selected
+  // and focused states for block labels
+  const $blockLabels = $(".block-label input[type='radio'], .block-label input[type='checkbox']");
+  // eslint-disable-next-line no-new
+  new GOVUK.SelectionButtons($blockLabels);
+});
+
+
+// add-a-job page
+$(document).ready(() => {
```
Comment (gjvis) [inline assets/js/main.js]: Same as the previous comment, no need to wait for document.ready I think

File: assets/stylesheets/style.scss [hunk 3]
```diff
@@ -1,4 +1,5 @@
 @import '_govuk-elements';
+@import "star-radios";
```
Comment (gjvis) [inline assets/stylesheets/style.scss]: inconsistent quotes (double vs single)

File: package.json [hunk 4]
```diff
@@ -37,8 +38,10 @@
     "gulp-uglify": "^1.5.3",
     "gulp-util": "^3.0.7",
     "hogan.js": "~3.0",
+    "jquery": "^2.2.4",
     "knex": "^0.11.5",
     "method-override": "~2.3",
+    "moment": "^2.13.0",
```
Comment (gjvis) [inline package.json]: pegged to a specific version intentionally?

Comment (lukasz-kaniowski) [inline package.json]: I tend not to modify output from `npm i --save`

File: test/common/page_objects/AddJobPage.js [hunk 5]
```diff
@@ -5,13 +5,32 @@ const AddJobPage = function AddJobPage(browser) {
...
+      title: browser.field('#job-title').value,
+      employer: browser.field('#job-employer').value,
+      sourceType: browser.field('[name="sourceType"][checked]').value,
+      sourceUrl: browser.field('#job-sourceUrl').value,
+      rating: browser.field('[name="rating"][checked]').value,
+      deadline: browser.field('#job-deadline').value,
+    });
+  this.fillDeadline = deadline => browser.fill('#job-deadline', deadline);
+  this.fillTitle = title => browser.fill('#job-title', title);
+  this.submit = () => browser.pressButton('input[type=submit]');
```
Comment (gjvis) [inline test/common/page_objects/AddJobPage.js]: is there a reason that we've referred to some fields using IDs and others using `[name="foo"]` type selectors?

File: test/features/support/Fixtures.js [hunk 6]
```diff
@@ -2,6 +2,10 @@ const Fixtures = function Fixtures() {
   this.sampleJob = {
     title: 'Test job',
     employer: 'Test employer',
+    sourceType: 'online',
+    sourceUrl: 'http://blog.queensland.com/best-of/best-job-in-the-world-best-of/',
+    rating: 4,
```
Comment (gjvis) [inline test/features/support/Fixtures.js]: if `Fixtures` is explicitly meant to be canned form data then this should be a string

File: views/layouts/main.mustache [hunk 7]
```diff
@@ -23,7 +23,9 @@
   {{/content}}
 
   {{$bodyEnd}}
+    <script type="text/javascript" src="{{{ assetPath }}}js/jquery.min.js"></script>
+    <script type="text/javascript" src="{{{ assetPath }}}js/selection-buttons.js"></script>
     <script type="text/javascript" src="{{{ assetPath }}}js/main.js"></script>
-  {{/bodyEnd}}
+{{/bodyEnd}}
```
Comment (gjvis) [inline views/layouts/main.mustache]: accidental removal of indentation

File: test/common/page_objects/AddJobPage.js (hunk 5 above)
Comment (lukasz-kaniowski) [inline test/common/page_objects/AddJobPage.js]: good call

**Comment by gjvis** [comment]
👍

**Pull request merged**